integration/check_test.go

```diff
@@ -85,6 +85,13 @@ func (s *SkopeoSuite) TestNeedAuthToPrivateRegistryV2WithoutDockerCfg(c *check.C
 	assertSkopeoFails(c, wanted, "--tls-verify=false", "inspect", fmt.Sprintf("docker://%s/busybox:latest", s.regV2WithAuth.url))
 }
 
+func (s *SkopeoSuite) TestCertDirInsteadOfCertPath(c *check.C) {
+	wanted := ".*flag provided but not defined: -cert-path.*"
+	assertSkopeoFails(c, wanted, "--tls-verify=false", "inspect", fmt.Sprintf("docker://%s/busybox:latest", s.regV2WithAuth.url), "--cert-path=/")
+	wanted = ".*unauthorized: authentication required.*"
+	assertSkopeoFails(c, wanted, "--tls-verify=false", "inspect", fmt.Sprintf("docker://%s/busybox:latest", s.regV2WithAuth.url), "--cert-dir=/etc/docker/certs.d/")
```

Comment:
@runcom 

Hum, this really only tests that the option name is acceptable; with `--tls-verify=false` and a HTTP-only registry the certificates in `--cert-dir` do not really matter. So the test is almost misleading, I’m not sure that it is really better than no test to be honest.

Preferably we should fix the
> `FIXME: Set up TLS for the docker registry port instead of using "--tls-verify=false" all over the place.`
todos, and then any smoke test of `skopeo inspect` would naturaly be a test for `--cert-dir`.

OTOH I don’t think it is at all reasonable to block this PR on doing that work.